tests/Mmi/Filter/UrlTest.php

```diff
@@ -23,6 +23,7 @@ public function testFilter()
         $this->assertEquals('', (new Url)->filter(null));
         $this->assertEquals('', (new Url)->filter('-'));
         $this->assertEquals('ala-ma-kota', (new Url)->filter('Ala ma kota!'));
+        $this->assertEquals('dokument-v1-2-23', (new Url)->filter('Dokument v1.2.23'));
```

Comment:
Pytanie czy to nie pozamienia setek urli gdzieś zapisanych :sweat:

Reply:
stare urle już przeliczone pozostaną takie jak były, nowe będą miały nowe.